features/main/configurable.feature

```diff
@@ -60,20 +60,3 @@ Feature: Configurable resource CRUD
       "foo": "Foo"
     }
     """
-
-  @dropSchema
```

Comment:
Without knowing if the test you're removing is relevant, if you are removing it, this annotation should be moved to the previous one, it likely causes the testsuite to fail

Reply:
Here we go : https://github.com/api-platform/core/issues/1724 :')

Reply:
#1687 will fix that ;)